rootfs/api/management/commands/load_db_state_to_k8s.py

```diff
@@ -24,7 +24,12 @@ def handle(self, *args, **options):
         # deploy applications
         print("Deploying available applications")
         for application in App.objects.all():
-            rel = application.release_set.latest()
-            application.deploy(rel)
+            try:
+                rel = application.release_set.latest()
```

Comment:
add a check to see if build object is none if it's true then should do application.create()